### PR TITLE
[Enhancement] Optimized fast_path_match()

### PR DESCRIPTION
**Pull request by tom-pytel**
* Changed fast_path_match() to use Python regex instead of working one character at a time.
* Also pulled out invariant disable_patterns out of loop in plugins.install().

### CODE REVIEW

File: skywalking/utils/ant_matcher.py [hunk 1]
```diff
@@ -16,75 +16,24 @@
 #
 
 
-def fast_path_match(pattern: str, path: str):
```
Comment (kezhenxu94) [inline skywalking/utils/ant_matcher.py:19]: This is ported from a Java version, I've just run a simple benchmark test, regex is much more better then the original version, thanks

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Comment by tom-pytel** [comment]
Hey, can't seem to find a way to open an issue on the NodeJS agent so will ask here, having a problem running the test due to the following:
```
Step 1/5 : FROM nodejs-dependencies
    ✓ test /home/tom/src/revdebugjs/skywalking-nodejs/tests/plugins/http
Service 'server' failed to build: pull access denied for nodejs-dependencies, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
Any idea?

**Comment by wu-sheng** [comment]
Hi @tom-pytel we share the issue list in the main repo to make people easier, feel free to submit a discussion here, https://github.com/apache/skywalking/issues with project name as the prefix of the issue title. :)

**Pull request merged**